.github/workflows/build.yml

```diff
@@ -43,27 +43,18 @@ jobs:
          - debian:bookworm # 12
          - debian:bullseye-slim # 11
          - debian:bullseye # 11
+         - ubuntu:plucky # 25.04
          - ubuntu:noble # 24.04
          - ubuntu:mantic # 23.10
          - ubuntu:jammy # 22.04
          - ubuntu:focal # 20.04
-         - nvidia/cuda:12.3.1-base-ubuntu22.04
-         - nvidia/cuda:12.3.1-base-ubuntu20.04
-         - nvidia/cuda:12.2.2-base-ubuntu22.04
-         - nvidia/cuda:12.2.2-base-ubuntu20.04
-         - nvidia/cuda:12.1.1-base-ubuntu22.04
-         - nvidia/cuda:12.1.1-base-ubuntu20.04
-         - nvidia/cuda:11.8.0-base-ubuntu22.04
-         - nvidia/cuda:11.8.0-base-ubuntu20.04
-         - nvidia/cuda:11.7.1-base-ubuntu22.04
-         - nvidia/cuda:11.7.1-base-ubuntu20.04
-         - nvidia/cuda:11.6.2-base-ubuntu20.04
-         - nvidia/cuda:11.4.3-base-ubuntu20.04
-         - nvidia/cuda:11.3.1-base-ubuntu20.04
-         - nvidia/cuda:11.2.2-base-ubuntu20.04
+         - nvidia/cuda:12.6.3-base-ubuntu24.04
+         - nvidia/cuda:12.6.3-base-ubuntu22.04
+         - nvidia/cuda:12.6.3-base-ubuntu20.04
+         - nvidia/cuda:11.6.1-base-ubuntu20.04
```

Comment:
i only used the latest supported cuda mentioned in https://hub.docker.com/r/nvidia/cuda